packages/node/src/config.js

```diff
@@ -45,6 +50,14 @@ module.exports = {
     },
     message: 'should be a function',
     validate: value => typeof value === 'function'
+  },
+  enabledBreadcrumbTypes: {
+    defaultValue: () => null,
+    message: `should be null or a list of available breadcrumb types (${NODE_BREADCRUMB_TYPES.join(',')})`,
+    validate: value => value === null || (isArray(value) && reduce(value, (accum, maybeType) => {
+      if (accum === false) return accum
+      return includes(NODE_BREADCRUMB_TYPES, maybeType)
+    }, true))
```

Comment:
This is overridden from the base config because the breadcrumb types will be different for node (I presume)

Reply:
I think they'd be the same actually — even if we don't use them in plugins now, we might in future so it makes sense to me to allow users to configure them up front

Reply:
Yeah good point, in which case I can remove this and leave it to pick up the default config